wordpress-mobile/AztecEditor-iOS
# Fixes a bug that was causing certain HTML contents to be lost.

### Description:

Fixes #1078 

This issue provides a mechanism for making sure data isn't lost on HTML elements without actual contents.

### Testing:

**Test 1:**
Run the newly added unit test.

**Test 2:**

As reported [here](https://github.com/wordpress-mobile/AztecEditor-iOS/issues/1078).

1. Open the empty Gutenberg demo.
2. Paste the following HTML content:
```html

<div style="height:100px" aria-hidden="true" class="wp-block-spacer"></div>

```
3. Switch to visual and back.

Make sure the content is still there.
